vnext/include/ReactUWP/IReactInstance.h

```diff
@@ -37,6 +37,7 @@ struct ReactInstanceSettings {
   bool EnableByteCodeCaching{false};
   bool EnableDeveloperMenu{false};
 
+  std::string AsyncLocalStorageDBPath;
```

Comment:
>AsyncLocalStorageDBPath [](start = 14, length = 23)

It does not seem right to add native module specific data to the common ReactInstanceSettings.

Reply:
I agree. What is your suggestion for how to avoid this?

Reply:
Where this setting is coming from? What components suppose to set it?

Reply:
It comes from the app. The app sets it.

Reply:
There are two ways that RN modules usually do this.  Either make it a parameter on the NativeModule ctor and then the app can create the nativemodule (or a custom factory or whatever) and pass that into the instance.  This of course means that the native module can't be hidden inside the core, but should actually be passed into the core. -- So a little more complex to setup, but more flexible.  -- It also would allow say office to pass in its own implementation of AsyncStorage, rather than use the one in core.

The other thing some modules do is just expose a static method to configure native modules.  Which would I guess be expose AsyncStorageNativeModule::SetStorageDBPath().  This of course has problems if you have multiple instances in an app, and want them to use different paths.

Reply:
I'll use a static method for now; having a ctor parameter would be better but we'll leave that to when this is out of core.